src/website/src/pages/eu/about/vision/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Vision
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+## Your gateway to the European Union
```

Comment:
I see you set the "subtitle" as in the text-box in the wiki, while the mock is different, confusing.. as it is confusing to see in mobile a different heading for the section in the "specs" but yeah, guess you're right :)